### DISCUSSION TITLE
No readings from HA

### DISCUSSION THREAD

**Discussion by VladCocan** [wmbusmeters/wmbusmeters#1404] [General]
i have 3 iPERL watermeters, but in 4 days, no telegram received.
HA is running in virtual machine, with usb passthrought
here are the logs :
cand somebody help me ?

(wmbusmeters) using log file /dev/stdout
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/nginx.sh
cont-init: info: /etc/cont-init.d/nginx.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun nginx (no readiness notification)
s6-rc: info: service legacy-services successfully started
[08:53:49] INFO: CONFIG_CONF ...
[08:53:49] INFO: {"device":"/dev/ttyACM0:iu891a:c1,t1","donotprobe":"/dev/ttyAMA0","format":"json","logfile":"/dev/stdout","loglevel":"debug","logtelegrams":false,"shell":"/wmbusmeters/mosquitto_pub.sh \"wmbusmeters/$METER_NAME\" \"$METER_JSON\""}
[08:53:49] INFO: CONFIG_METERS ...
[08:53:49] INFO: []
[08:53:49] INFO: Syncing wmbusmeters configuration ...
cp: can't stat '/data/drivers/*': No such file or directory
[08:53:49] INFO: Registering meters ...
[08:53:49] INFO: Generating MQTT configuration ... 
[08:53:50] INFO: Broker core-mosquitto will be used.
[08:53:50] INFO: 
MQTT Discovery ...
[08:53:50] INFO: MQTT Discovery cleanup...
[08:53:51] INFO: Starting web configuration service.
[08:53:51] INFO: Running wmbusmeters ...
(config) "logtelegrams" "false"
(wmbusmeters) version: 1.17.1-29-g3af7a4c
(config) using device: /dev/ttyACM0:iu891a:t1,c1 
(config) number of meters: 0
(lookup) with file/hex "/dev/ttyACM0" iu891a
(serial) check if /dev/ttyACM0 can be accessed
(serial) tty /dev/ttyACM0 can be accessed
(serial) before config: /dev/ttyACM0 speed(B115200) input() output() control(CLOCAL|CREAD|CSIZE) local() special_chars(58,50,27,103,57,40,94,34) modem(DTR|RTS|CTS)
(serial) after config:  /dev/ttyACM0 speed(B115200) input() output() control(CLOCAL|CREAD|CSIZE) local() special_chars(58,50,27,103,16) modem(DTR|RTS|CTS)
(serialtty) opened /dev/ttyACM0 fd 3 (detect iu891a)
(serial) received binary ""
(serial /dev/ttyACM0) sent "C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0C0"
(serial /dev/ttyACM0) sent "C001030424C0"
(serial) EVENT thread interrupted
(serial) received binary "C00104006E83050000BE2D060041040000A1E7C0"
(iu891a) device info response "6E83050000BE2D060041040000"
(iu891a) endpoint 01 msg 04 status 00
(iu891a) info:  type=iu891a uid=00000583
(serial /dev/ttyACM0) sent "C00981DE4DC0"
(serial) EVENT thread interrupted
(serial) received binary "C0098200B32500201089010EA9E0C0"
(iu891a) wmbus address response "B32500201089010E"
(serialtty) closed /dev/ttyACM0 (detect iu891a)
(serial) EVENT thread interrupted
(iu891a) are you there? yes 00202089
(main) opening /dev/ttyACM0:iu891a:t1,c1
Started config iu891a[00202089] on /dev/ttyACM0 listening on t1,c1
(iu891a) on /dev/ttyACM0
(serial) EVENT thread interrupted
(serial) before config: /dev/ttyACM0 speed(B115200) input() output() control(CLOCAL|CREAD|CSIZE) local() special_chars(111,50,27,103,57,40,94,34) modem(DTR|RTS|CTS)
(serial) after config:  /dev/ttyACM0 speed(B115200) input() output() control(CLOCAL|CREAD|CSIZE) local() special_chars(111,50,27,103,16) modem(DTR|RTS|CTS)
(serialtty) opened /dev/ttyACM0 fd 3 (iu891a)
(iu891a) get device info
(serial /dev/ttyACM0) sent "C001030424C0"
(serial) EVENT thread interrupted
(serial) received binary "C00104006E83050000BE2D060041040000A1E7C0"
(iu891a) rsp got device info
(iu891a) device info:  type=iu891a uid=00000583
(iu891a) get wmbus address
(serial /dev/ttyACM0) sent "C00981DE4DC0"
(serial) received binary "C0098200B3250020"
(iu891a) partial frame, expecting more. "C0098200B3250020"
(serial) received binary "1089010EA9E0C0"
(iu891a) rsp got wmbus address
(iu891a) device info: 00202089  type=iu891a uid=00000583
(iu891a) get config
(serial /dev/ttyACM0) sent "C00901D6C9C0"
(serial) EVENT thread interrupted
(serial) received binary "C0090200"
(iu891a) partial frame, expecting more. "C0090200"
(serial) received binary "030E000000320010270000E8CFC0"
(iu891a) rsp got active config
(iu891a) config: RCV_ALL RCV_IND SND_IND RECALIB led flash: 50 ms recalibrate: 10000 ms link modes: t1,c1
(iu891a) got device id 00202089
(main) regular reset of iu891a  on /dev/ttyACM0 will happen every 82800 seconds
(iu891a) set config
(serial /dev/ttyACM0) sent "C00903030E000000320010270000919CC0"
(serial) EVENT thread interrupted
(serial) received binary "C00904"
(iu891a) partial frame, expecting more. "C00904"
(serial) received binary "00B23DC0"
(iu891a) rsp set active config
(iu871a) set config to set link mode 03
(wmbus) no alarm (expected activity) for iu891a
(serial) registered regular callback HOT_PLUG_DETECTOR(0) every 2 seconds
No meters configured. Printing id:s of all telegrams heard!
(serial) waiting for stop
[08:53:51] INFO: Starting Nginx...
2024/10/25 08:53:51 [notice] 83#83: using the "epoll" event method
2024/10/25 08:53:51 [notice] 83#83: nginx/1.24.0
2024/10/25 08:53:51 [notice] 83#83: OS: Linux 6.6.54-haos
2024/10/25 08:53:51 [notice] 83#83: getrlimit(RLIMIT_NOFILE): 1073741816:1073741816
2024/10/25 08:53:51 [notice] 83#83: start worker processes
2024/10/25 08:53:51 [notice] 83#83: start worker process 210

**Comment by BIBOLV** [comment]
From quick look on logs - everything looks fine. Are you sure that your meters are transmitting wmbus messages?
